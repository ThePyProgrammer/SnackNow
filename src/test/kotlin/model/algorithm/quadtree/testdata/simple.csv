0,0.01,0.01,0
4,1
0.001,0.001
0.003,0.003
0.008,0.004
0.008,0.004
0,0.01,0.005,0
