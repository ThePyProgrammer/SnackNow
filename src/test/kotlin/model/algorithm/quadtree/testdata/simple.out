1) Queried (0.00000, 0.01000) to (0.00500, 0.00000) and received 2 result(s):
[(0.00100, 0.00100), (0.00300, 0.00300)]